src/index.html

```diff
@@ -1009,6 +1009,23 @@ <h2 class="app__title">
 			</style>
 		</a>
 
+		<a class="list__item app js-app"
+				href="https://resilientwebdesign.com/"
+				data-app="ResilientWebDesign"
```

Comment:
Okay, done (but there are some existing entries where the app IDs and icon file names don't match up).